Cargo.toml

```diff
@@ -97,6 +97,7 @@ thread-priority = "0.16.0"
 tokio = { version = "1.36.0", features = ["fs", "time"] }
 tracing = "0.1.40"
 tracing-subscriber = "0.3.18"
+sp-api = "28.0.0"
```

Comment:
Every single dependency until this point was sorted alphabetically, please preserve that.

Reply:
Fixed in commit@https://github.com/subspace/space-acres/commit/bd5fbd77b7ebd8b8073c93562d2ac8437f11f5ee